tests/wpt/meta/css/CSS2/floats-clear/floats-149.xht.ini

```diff
@@ -0,0 +1,2 @@
+[floats-149.xht]
```

Comment:
I think this shouldn't be regressing, it's quite strange that two cells with `width: 50%` end up having a different width.

```html
<style>
table {
  width: 200px;
  border-spacing: 0px;
}
td {
  padding: 50px 0px;
  width: 50%;
  background: linear-gradient(to right, cyan, magenta);
}
</style>
<table>
  <tr><td></td><td></td></tr>
</table>
```